src/fromJEOL.js

```diff
@@ -83,9 +81,10 @@ export function fromJEOL(buffer) {
     if (info.dataUnits[d] === 'Second') {
       options.quantityName = 'time';
       options.originOffset = { magnitude: 0, unit: 's' };
+
       if (d === 0) {
         options.coordinatesOffset = {
-          magnitude: info.digitalFilter * increment,
+          magnitude: info.digitalFilter * increment.magnitude,
```

Comment:
this should be set as an array

Reply:
this options is for each dimension, should it be a 1d array?